.ci/jobs/package-registry.yml

```diff
@@ -0,0 +1,23 @@
+---
+---
+- job:
+    name: Beats/build-it-docker-images
+    display-name: ITs Docker images
+    description: Job to pre-build docker images used in integration tests.
+    view: Beats
+    project-type: pipeline
+    pipeline-scm:
+      script-path: .ci/build-docker-images.groovy
+      scm:
+        - git:
+            url: git@github.com:elastic/beats.git
+            refspec: +refs/heads/*:refs/remotes/origin/* +refs/pull/*/head:refs/remotes/origin/pr/*
+            wipe-workspace: 'True'
+            name: origin
+            shallow-clone: true
+            credentials-id: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+            reference-repo: /var/lib/jenkins/.git-references/beats.git
+            branches:
+              - 'master'
+    triggers:
+      - timed: 'H H(0-5) * * 1-5'
```

Comment:
Is this the same as the triggers in the groovy file?

Reply:
yep, you have to define it, if not the job will not be scheduled until you run it manually. It is a long history, but Jenkins files are not loaded until you execute the job for the first time, so everything you define there does not exists until you execute the file once.